pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -270,8 +270,8 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 			return
 		}
 		// create list of ips and gateways
-		cidr.IP[len(cidr.IP)-1] += 1 // Set bridge address to first address in IPNet
 		plugin.podCIDRs = append(plugin.podCIDRs, cidr)
+		cidr.IP[len(cidr.IP)-1] += 1 // Set bridge address to first address in IPNet
```

Comment:
AFAIU `cidr` is `*net.IPNet`, and IP is a byte slice, and doing this will still affect the object in the `plugin.podCIDRs`. I was thinking about smth like:
```
plugin.podCIDRs = append(plugin.podCIDRs, cidr)
gwAddr := make(net.IP, len(cidr.IP))
copy(gwAddr, cidr.IP)
gwAddr[len(gwAddr) - 1] += 1
plugin.podGateways = append(plugin.podGateways, gwAddr)
```

Reply:
how is affecting plugin.podCIDRs?

Reply:
> AFAIU `cidr` is `*net.IPNet`, and IP is a byte slice, and doing this will still affect the object in the `plugin.podCIDRs`.
I see your point, I think you are right

Reply:
@baranov1ch do you want to submit the PR to fix it?
I was thinking you won't work on it, I'm happy to close this and let you work on it